src/OpenFeature.DependencyInjection/OpenFeatureBuilder.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.Extensions.DependencyInjection;
+
+namespace OpenFeature;
+
+/// <summary>
+/// Describes a <see cref="OpenFeatureBuilder"/> backed by an <see cref="IServiceCollection"/>.
+/// </summary>
+/// <param name="Services">The <see cref="IServiceCollection"/> instance.</param>
+public sealed record OpenFeatureBuilder(IServiceCollection Services)
+{
+    /// <summary>
+    /// Indicates whether the evaluation context has been configured.
+    /// This property is used to determine if specific configurations or services
+    /// should be initialized based on the presence of an evaluation context.
+    /// </summary>
+    internal bool IsContextConfigured { get; set; }
```

Comment:
Just to be sure: why is this `internal`? 
If our package benefits from `IsContextConfigured` could it be beneficial for external extensions too?

Reply:
Great question! I initially made it internal because it was primarily used for internal configuration, and the public setter was intended for use within the internal scope. However, you raise a good point. There's no strong reason not to make it public, as it could provide useful information for consumers. We can certainly change it to:

`public bool IsContextConfigured { get; internal set; }`

Reply:
Updated